.github/workflows/test-and-lint.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       shell: bash -l {0}
       run: |
         conda install readdy==2.0.9
+        conda install numpy
```

Comment:
should this have the same version specifier that I see in setup.py's requirements?